pod-operation/src/components/wheel_encoder.rs

```diff
@@ -38,7 +38,7 @@ impl WheelEncoder {
 			self.counter += 1.0;
 
 			let current_time = Instant::now();
-			let distance = (self.counter * 5.0) / 1000.0;
+			let distance = (self.counter / 18.0) * 3.28084;
```

Comment:
Question: could you elaborate where these magic values come from? The encoder we're using has 16 pulses per cycle (not 18), and the wheel diameter we used last year was 3 inches.

Reply:
testing values not accurate, will change now thx